src/components/side_panel/custom_currency/custom_currency.xml

```diff
@@ -51,6 +51,23 @@
             />
           </t>
         </select>
+        <t t-set="accounting_format_label">Accounting format</t>
+        <Checkbox
+          name="'accountingFormat'"
+          label="accounting_format_label"
+          value="state.isAccountingFormat"
+          onChange="() => this.toggleAccountingFormat()"
```

Comment:
nitpick `onChange.bind="toggleAccountingFormat"` (the anonymous function can lead to potentially useless renders since the props is always referencialy different at each render)